app/models/govbox/message.rb

```diff
@@ -25,7 +25,7 @@ class Govbox::Message < ApplicationRecord
 
   DELIVERY_NOTIFICATION_TAG = 'delivery_notification'
 
-  def self.create_message_with_thread!(govbox_message)
+  def self.create_message_with_thread!(govbox_message, priority: :default)
```

Comment:
Tu sme si @jsuchal povedali, nech si radsej posielame kontext do tejto metody ako prioritu (a vo vnutri metody to prelozime na prioritu). Vidim vsak v tom trochu problem, lebo kontext moze byt rozny. Inicialny import (nizka priorita), stahovanie spravy do vlast. ruk po prevzati dorucenky (vysoka priorita) alebo ziadny kontext (default priorita). 
A tento kontext nemame velmi ako v ProcessMessageJobe (ktory vola tuto metodu) poznat, museli by sme si ho tam posielat napriec viacerymi jobmi, aby vedel prebublat az sem.